components/proof-of-yield/ChartDetailsButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { PropsWithChildren } from "react";
+import Image from "next/image";
+import { GradientButton } from "../../components";
+import { useViewWidth } from "../../hooks";
+import { lgSize } from "../../constants";
+import { assetRootPath } from "../../utils";
+
+interface ChartDetailsButtonProps {
+  onClick: () => void;
+}
+
+const ChartDetailsButton = ({
+  onClick,
+  children,
+}: PropsWithChildren<ChartDetailsButtonProps>) => {
+  const width = useViewWidth();
```

Comment:
This is probably fine but we should try to just be consistent with using this and tailwind breakpoints. Are they set up the same?

Reply:
Yup, they are